config/docker/gcc-10_arm64/Dockerfile

```diff
@@ -2,6 +2,7 @@ ARG PREFIX=kernelci/
 FROM ${PREFIX}build-base
 
 RUN apt-get update && apt-get install --no-install-recommends -y \
+    dwarves \
```

Comment:
Why is this only for arm64?

Reply:
Will do.
Just didnt setup yet local build environment for x86, planned to add to more compilers as soon as i complete local tests.